build.sh

```diff
@@ -26,14 +26,26 @@ if [ -z "$build_image_name" ]; then
 	exit 1
 fi
 
-if [ $(DOCKER_CLI_EXPERIMENTAL=enabled docker manifest inspect $build_image_name > /dev/null ; echo $?) -eq 0 ]; then
+if [ $(DOCKER_CLI_EXPERIMENTAL=enabled docker manifest inspect $build_image_name 2> /dev/null ; echo $?) -eq 0 ]; then
 	echo "----> Skipping build as image already exists"
 else
-	pack build $build_image_name \
```

Comment:
These changes are for the AWS credentials issue, nothing in this file is related to the PR title.